src/span.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+import * as Constants from './constants';
```

Comment:
I'm not a JS expert, but most of our internal Node code I've seen uses `require`, I've never seen imports like this. Is it really universal syntax?

cc @oibe